applications/sofa/gui/GUIManager.cpp

```diff
@@ -206,6 +206,30 @@ int GUIManager::Init(const char* argv0, const char* name)
         first = false;
     }
 
+    if (sofa_prefix.empty())
```

Comment:
@guparan you are our expert in deployement and filesytem...what is your opinion onthe following ?

Reply:
I think this is not needed. Paths contained in sofa.ini are supposed to be added to DataRepository at construction time. @jnbrunet why did you need this?

Reply:
This was done to allow one to change the sofa prefix manually. Without this, the only way was to set the environment variable SOFA_ROOT. Now this adds a way to do it in a cpp code or python code with the proper binding code.

Reply:
Hum, I think the code for reading the ini file have been moved since I did this code... Let me verify that